index.rst

```diff
@@ -1,57 +1,240 @@
+the M1 pressure in the ATPneumatics depending on the elevation. It works by
+listening to the position of telescope, published by the ATMCS, and the current
+M1 support pressure, published by the ATPneumatics. The elevation is used to
+calculate the expected pressure, according to a polynomial equation. If the
+computed value changes by more than a configurable threshold compared to the
+current pressure, it will send a command to the ATPneumatics to update the M1
+support pressure.
+
+This document describes the procedure to measure the optimum M1 pressure for
+different telescope elevations and fit the data with a polynomial equation
+to obtain the coefficients required by the ATAOS component. We also describe
+how to load the new coefficients into the ATAOS.
+
+
+Measuring optimum M1 support pressure for different elevations
+==============================================================
+
+.. important::
+   During this process it is important to make sure one does not lift the
+   mirror from the hard points. This will happen if the value measured by
+   the load cell gets close to zero. In this case, it is important to lower
+   the mirror all the way back, setting the pressure to zero, close the
+   M1 valve in the ATPneumatics, slew the telescope to the park position
+   (:math:`\mathrm{elevation} = 86^\circ`), and restart the process.
+
+As mentioned above, the support system contain 3 hard points and 21 actuators,
+all have the same contact area with the mirror. One of the hard points (the
+load cell) contain a "digital scale" that measures the load (how much weight)
+it is supporting.
+
+In principle, to minimize distortion in the mirror shape we must make sure
+the load is evenly distributed between all the 24 actuators. The first step in
+the process is determining what it expected "optimum load", as measured by the
+hard point load scale mentioned above. When there is no pressure in the
+pneumatic system all the weight of the mirror will be supported by the 3 hard
+points. That means the "total load" for a given elevation is 3x the one
+measured by the load cell. Consequently, the "optimum load" will be the "total
+load" divided by 24, the number of actuators.
+
+.. math:: \mathrm{optimum\_load} = \frac{\mathrm{load\_cell} \times 3}{24}
+   :label: eq-optimum-load
+
+Once the optimum load is defined for a given elevation we need to determine
+what is the corresponding pressure set point to the ATPneumatics needed.
+In order to obtain the value for the optimum pressure we then perform a
+binary search inside a sensible range.
 
-  and
+.. note::
+   As the pressure increases in the M1 support system the load decreases.
 
-  Subsections
-  -----------
+Assuming a lower and upper limit for the pressure of :math:`P_{min}` and
+:math:`P_{max}`, the binary search for the optimum pressure is as follows:
 
-  and
+   1. Set pressure to zero (:math:`P=0`) and measure load to determine
```

Comment:
I don't see any mention of the mass of the mirror. It might be worth mentioning it somewhere and then how the technique doesn't actually require it.